Makefile

```diff
@@ -34,6 +34,10 @@ test-e2e-prepared-node:
 build-tools:
 	cd ./tools/windows-node-installer && $(GO_BUILD_ARGS) go build -o wni $(TOOLS_DIR)
 
+.PHONY: test-unit-tools
+test-unit-tools:
```

Comment:
Please call this `test-unit-wni`. I am going with that sort of naming in https://github.com/openshift/release/pull/8623 to clearly call our what we are testing. Yes it is going against the strange naming we have in place which we should change.

Reply:
Done